calicoctl/calicoctl/calicoctl.go

```diff
@@ -40,7 +40,7 @@ func main() {
     replace      Replace a resource by file, directory or stdin.
     apply        Apply a resource by file, directory or stdin.  This creates a resource
                  if it does not exist, and replaces a resource if it does exists.
-    patch        Patch a pre-exisiting resource in place.
+    patch        Patch a preexisting resource in place.
```

Comment:
`preexisting` has entered standard use. If people want to retain the `-`, that's fine, but the typo (which was removed from the docs) should at least be fixed.